docroot/themes/custom/mass_admin_theme/js/auto-complete-combobox.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * @file
+ * Add missing combobox association between auto complete fields and
+ * their option boxes with aria.
+ *
+ * This addresses the accessibility issue with JAWS:  DP-25848
+ */
+(function () {
+  'use strict';
+
+  // The timeout function is necessary to recognize the fields and the lists.
+  setTimeout(function () {
```

Comment:
@ygannett Did you happen to try an event listener here? Something like DOMContentLoaded? Also, it could maybe make sense to wrap this in a Drupal.behavior, which would act on the page when the DOM is ready.

Reply:
All timeout functions are replaced.